modules/social_features/social_event/modules/social_event_enrollment_max/social_event_max_enroll.module

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @file
+ * The Social Event Max Enroll module.
+ */
+
+use Drupal\views\ViewExecutable;
+use Drupal\views\Plugin\views\cache\CachePluginBase;
+use Drupal\node\Entity\Node;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_views_post_render().
+ *
+ * Alter "Event enrollments" views. Add number of places left .
+ */
+function social_event_max_enroll_views_post_render(ViewExecutable $view, &$output, CachePluginBase $cache) {
+  if ($view->id() == 'event_enrollments' && $view->current_display == 'event_enrollments') {
+    if (isset($output['#rows'])) {
+      if (!empty($view->args[0])) {
+        $nid = $view->args[0];
+        $node = Node::load($nid);
+
+        // Add places left counter to the block title.
+        if (social_event_max_enroll_is_enabled($node)) {
+          // Count how many places left.
+          $left = social_event_max_enroll_left($node);
+
+          if ($left > 0) {
+            $title_suffix = \Drupal::translation()->formatPlural($left, '(1 place left)', '(@count places left)');
+          }
+          else {
+            $title_suffix = t('(0 places left)');
+          }
+
+          $view->header['result']->options['content'] .= ' ' . $title_suffix;
+        }
+
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_form_alter().
+ *
+ * Check and display "Allow anonymous enrollments" checkbox for public events.
+ */
+function social_event_max_enroll_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $event_forms = [
+    'node_event_form',
+    'node_event_edit_form',
+  ];
+
+  if (in_array($form_id, $event_forms)) {
+    $config = \Drupal::config('social_event.settings');
+    // Hide checkbox if feature is disabled globally.
+    $form['field_event_max_enroll']['#access'] = $config->get('max_enroll');
```

Comment:
Doesn't work on installation, probably because no default setting is set.

![screenshot 2019-01-03 at 14 51 18](https://user-images.githubusercontent.com/7124754/50641103-35319200-0f67-11e9-95f8-3f35f0b65018.png)

![screenshot 2019-01-03 at 14 51 21](https://user-images.githubusercontent.com/7124754/50641112-39f64600-0f67-11e9-8290-bdf214ec56bc.png)